package.json

```diff
@@ -38,5 +38,11 @@
     "webpack": "^4.42.1",
     "webpack-cli": "^3.3.11",
     "webpack-dev-server": "^3.11.0"
+  },
+  "dependencies": {
+    "@gnosis.pm/safe-contracts": "^1.1.1-dev.2",
```

Comment:
Do we need this package? If we are just using this to get GnosisSafe ABI `import GnosisSafeSol from "@gnosis.pm/safe-contracts/build/contracts/GnosisSafe.json";`, we can directly copy into our ABI folder, right?

Reply:
As it upgradeable so it could change. By using the package we don't need to maintain it.